### PR TITLE
Setup basic middlewares

### PR DESCRIPTION
**Pull request by joycinho26**
Creation of Middlewares

### CODE REVIEW

File: BankingSystem.API.Tests/BankingSystem.API.Tests.csproj [hunk 1]
```diff
@@ -16,6 +16,8 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
+    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerGen" Version="6.5.0" />
```
Comment (vliashko) [inline BankingSystem.API.Tests/BankingSystem.API.Tests.csproj]: Swagger not needed in Tests projects

File: BankingSystem.API/.BankingSystem.API.csproj.swp
Comment (vliashko) [inline BankingSystem.API/.BankingSystem.API.csproj.swp:1]: For what this file?

File: BankingSystem.API/Extensions/ApplicationDependanciesConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+{
+    public static class ApplicationDependanciesConfiguration
+{
+    public static IServiceCollection ConfigureServices(this WebApplicationBuilder builder)
+    {
+        builder.Services.AddSwaggerGen();
+        builder.Services.AddAuthorization();
+        builder.Services.AddLogging(options =>
+        {
+            options.AddConsole();
```
Comment (vliashko) [inline BankingSystem.API/Extensions/ApplicationDependanciesConfiguration.cs]: Please add Serilog library for our logging implementation

File: BankingSystem.API/Program.cs [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+
+app.UseCors(options =>
+{
+    options.AllowAnyOrigin()
+           .AllowAnyMethod()
+           .AllowAnyHeader();
+});
+
+app.UseMiddleware<GlobalExceptionHandlerMiddleware>();
+app.UseAuthorization();
```
Comment (vliashko) [inline BankingSystem.API/Program.cs:22]: UseAuthentication is also needed

**Review by vliashko — CHANGES_REQUESTED** [review]

File: BankingSystem.API.Tests/BankingSystem.API.Tests.csproj (hunk 1 above)
Comment (joycinho26) [reply BankingSystem.API.Tests/BankingSystem.API.Tests.csproj]: Fixed

File: BankingSystem.API/.BankingSystem.API.csproj.swp
Comment (joycinho26) [reply BankingSystem.API/.BankingSystem.API.csproj.swp:1]: Fixed

File: BankingSystem.API/Extensions/ApplicationDependanciesConfiguration.cs (hunk 2 above)
Comment (joycinho26) [reply BankingSystem.API/Extensions/ApplicationDependanciesConfiguration.cs]: Added

File: BankingSystem.API/Program.cs (hunk 3 above)
Comment (joycinho26) [reply BankingSystem.API/Program.cs:22]: Added

**Review by vliashko — APPROVED** [review]

**Pull request merged**